pkg/kubelet/rkt/rkt.go

```diff
@@ -1078,8 +1113,8 @@ func (r *Runtime) convertRktPod(rktpod *rktapi.Pod) (*kubecontainer.Pod, error)
 			// By default, the version returned by rkt API service will be "latest" if not specified.
 			Image:   fmt.Sprintf("%s:%s", app.Image.Name, app.Image.Version),
 			Hash:    containerHash,
-			Created: podCreated,
 			State:   appStateToContainerState(app.State),
+			Created: rktpod.CreatedAt,
```

Comment:
Good call, though I'm pretty sure that field isn't read anywhere. I'll update it appropriately and also do a PR to remove it entirely separately.